lib/streamlit/caching.py

```diff
@@ -438,12 +439,16 @@ def cache(
     ...     return data
 
 
-    To override the default hashing behavior, pass a mapping of type to hash function:
+    To override the default hashing behavior, pass a mapping of type or fqn to hash function:
```

Comment:
This would be clearer:

```
To override the default hashing behavior, pass a custom hash function. You can do that by mapping a type (e.g. `MongoClient`) to a hash function (`id`) like this:

(Example 1)

Alternatively, you can map the type's fully-qualified name (e.g. `"pymongo.mongo_client.MongoClient"`) to the hash function instead:

(Example 2)
```